services/api/src/typeDefs.js

```diff
@@ -518,6 +505,10 @@ const typeDefs = gql`
     """
     name: String
```

Comment:
should we change this to `displayName`? As this suggests more that this is now a name for humans and could actually change?

Reply:
and keep `uniqueName` or rename that to `name`? changing what `name` returns is a potentially more confusing break than renaming it though

Reply:
I would have sunsetted `name` all together and just use `uniqueName` and `displayName`, we could still have `name` as an attribute on the Environment and just return the `displayName` for now, but mark it as deprecated?

Reply:
Update pushed up including uniqueName and displayName